probe_src/arena/arena.c

```diff
@@ -17,6 +17,11 @@
 #include <string.h>
 #include <sys/mman.h>
 #include <fcntl.h>
+#include <assert.h>
+#include <unistd.h> 
+#include <sys/stat.h>
+#define memcpy_s(dest, destsz, src, count)
+#define snprintf_s(buffer, sizeOfBuffer, count, ...)
```

Comment:
What is the point of these two macros?

Reply:
`clang-tidy` was giving warnings that `memcpy` and `snprintf` aren't secure, so to remove those added `memcpy_s` and `snprintf_s`. These two aren't defined on non-window environments so had to define them as well

Reply:
It seems like you're defining them as nothing "replace `memcpy_s(a, b, c, d)` with the empty-string".

Maybe we should define `memcpy_s(a, b, c, d)` as `memcpy(a, c, d)`?